src/java.base/share/classes/sun/security/tools/keytool/Main.java

```diff
@@ -2170,6 +2194,10 @@ private void doPrintEntry(String label, String alias, PrintStream out)
                 if (verbose || rfc || debug) {
                     out.println(rb.getString
                         ("Certificate.chain.length.") + chain.length);
+
+                    X509Certificate[] xcerts = convertCerts(chain);
```

Comment:
I got `java.lang.ClassCastException` when casting `Certificate[]` array form here. However, I should cast Certificate object directly instead of reparsing the certificate, and made the change to do so.